forum/communications/mongo_models.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from mongoengine import Document, EmbeddedDocument, fields, CASCADE
+
+class NamespaceInfo(EmbeddedDocument):
+    user_id = fields.LongField(required=True)
+    namespace = fields.StringField(required=True)
+    namespace_id = fields.LongField(required=True)
+
+
+class Notification(Document):
+    initiator = fields.EmbeddedDocumentField(NamespaceInfo, required=True)
+    receivers = fields.EmbeddedDocumentListField(NamespaceInfo, required=True)
+    message = fields.StringField(required=True, max_length=255)
+    created_at = fields.DateTimeField(default=datetime.now)
```

Comment:
Use `datetime.utcnow` instead of `datetime.now` to avoid potential timezone issues

Reply:
Changed to using the datetime.utcnow method, but for the record, this method is deprecated in Python 3.12 (it might be removed or replaced with something else later)